spec/controllers/api/public/v1/providers/courses_controller_spec.rb

```diff
@@ -60,5 +60,37 @@
         expect(JSON.parse(response.body)["data"].size).to eql(2)
       end
     end
+
+    describe "include" do
+      let(:course) { create(:course, :with_accrediting_provider, provider: provider) }
+      before do
+        course
+      end
+
+      it "returns the provider and accrediting body connected to the course" do
+        get :index, params: {
```

Comment:
Could we put this `get` request in the before block so it's in the arrange/act/assert structure? So the it block is just doing the assertion